### ISSUE TITLE
how to draw conclusion？

### ISSUE THREAD

**Issue by dufree** [rdgarce/DDPG-design-for-lane-keeping-in-TORCS-Environment#1]
how to use  file of h5 or pkl to generate graphs. i want to see the loss or iterative process while training. thanks

**Comment by rdgarce** [comment]
I have not included any pre-trained model so you won't find any h5 files.

If you want to start a training you must have TORCS installed with SCR patch and head to Code/Client.py and execute the script with all the default parameters defined on top of it. (Maybe you will need to change the PORT of the connection if you use one different from 3001). When training, after each episode you will find a new folders in DDPG/models. Each new folder is the model saved at each episode.

In this work i'm not using any of the OpenAi-gym-like TORCS so the evnironment is taken care in the Client script. If you want to print the reward function you can uncomment like 98.
If you want to print the losses of the actor/critic you must modify agent.train(), in which losses are calculated.

I'm sorry for the poor readable code but this is one of my first project...
In the future i'm planning to revise the code and improve readability.
Hope this comment helps you

**Comment by dufree** [comment]
Ahaha you have made a completed project. i just want to run the project and test my own RL alg.  Even i haven't trained well but it helps me lot hhh.

**Comment by rdgarce** [comment]
Hello Andy,
I finally had time to edit the repo adding the "experiment" folder.
In it you can launch the "launch.py" and automatically load the model and try it in any TORCS race.
